src/components/LanguagePicker/LanguagePickerList.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {useState} from 'react'
+import {FlatList, StyleSheet, View} from 'react-native'
+
+import {LanguagePickerListItem} from './LanguagePickerListItem'
+import {LanguagePickerListSearchInput} from './LanguagePickerListSearchInput'
+
+export function LanguagePickerList({
```

Comment:
stick with the declaration style that is already established
if you want to change it, bring it up with the team
if we all agree, we should changed universally/atomically